specification/cognitiveservices/data-plane/LUIS/Runtime/readme.md

```diff
@@ -0,0 +1,35 @@
+# Cognitive Services LUIS SDKs
+
+> see https://aka.ms/autorest
+
+Configuration for generating LUIS Runtime SDK.
+
+``` yaml
+tag: runtime_2_0
+add-credentials: true
+openapi-type: data-plane
+```
+
+The current release for the Runtime Endpoint is `runtime_2_0`.
+
+# Releases
+
+## Runtime 2.0
+These settings apply only when `--tag=runtime_2_0` is specified on the command line.
+
+``` yaml $(tag) == 'runtime_2_0'
```

Comment:
``` yaml $(tag) == 'runtime_2_0'
input-file: v2.0/LUIS-Runtime.json

# remove Resolve2 from code-generation (since the POST and GET operations are functionally identical. )
directive:
  remove-operation: Prediction_Resolve2
 
```